.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
   - repo: https://github.com/pycqa/isort
-    rev: 5.9.0
+    rev: 5.11.5
```

Comment:
i also fixed this as part of some repo maintenance today, could you back this change out of this PR?

Reply:
Done!